api/v1alpha1/podiochaos_types.go

```diff
@@ -124,6 +131,32 @@ type AttrOverrideSpec struct {
 	Rdev *uint32 `json:"rdev,omitempty"`
 }
 
+// MistakeSpec represents one type of mistake
+type MistakeSpec struct {
+	// Filling determines what is filled in the miskate data.
+	// +optional
+	Filling FillingType `json:"filling,omitempty"`
+
+	// There will be [1, MaxOccurrences] segments of wrong data. If MaxOccurrences is 0, no mistake will be injected
+	// +optional
+	MaxOccurrences uint64 `json:"maxOccurrences,omitempty"`
+
+	// Max length of each wrong data segment in bytes
+	// +optional
+	MaxLength uint64 `json:"maxLength,omitempty"`
+}
```

Comment:
I just realize that kubernetes do not recommend use unsigned integer:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types.

PTAL @AsterNighT ;

Reply:
Oops, fresh knowledge to me, thanks!

Reply:
Oh that'll lead to a disaster. There're tons of uint16/32/64 in out code. What're we gonna do about that?

Reply:
I think just keep not using unsigned int in `api/v1alpha1`, and use webhook instead.

I try to count that, it is about 15- 20 refs to `uint` things, it's not so big.  🙈

Reply:
I've fixed my uints, but I'm not sure what it is for the others. 🤦